build.sbt

```diff
@@ -105,12 +105,15 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
 
 scalastyleConfig := baseDirectory.value / "project" / "scalastyle-config.xml"
 
+unmanagedResourceDirectories in Compile += baseDirectory.value / "public"
```

Comment:
can you move this line back to where it was? just to reduce noise in commit

Reply:
I moved it because it splits up the definition of the vars 'compileDependencies' and 'testDependencies' with their usage in the 'libraryDependencies'. Makes the code easier to follow.